.vale/styles/RedHat/TermsSuggestions.yml

```diff
@@ -22,6 +22,7 @@ swap:
   client side|client-side: client-side (adjective)| client side (noun)
   code: write
   crash: fail|lock up|stop|stop responding
+  Elastic Load Balancer: Elastic Load Balancing
```

Comment:
You only need this in one place. Suggest removing it from here and leaving it in the Errors rule.

Reply:
Removed